### ISSUE TITLE
Install hook may fail if there is already a stream block

### ISSUE THREAD

**Issue by josegonzalez** [rvanlaar/dokku-nginx-stream#1]
Not sure how we might detect an existing stream block, but thats one potential issue.

**Comment by rvanlaar** [comment]
Noted. Ran into this problem when testing.

Regular nginx configurations don't seem to include a stream block.

**Comment by josegonzalez** [comment]
Yep thats correct. My gut says that you can regex find an uncommented stream block in the conf and then add it if it isn't there.

[This library](https://github.com/lytics/confl) could be used to parse the conf files, but dunno how strict/not strict it needs to be.

**Comment by rvanlaar** [comment]
I'm rating this as a low priority issue. I've yet to see a default config that includes a stream directive.
When that happens, I'll be happy to work with those affected to include a fix.
